src/services/timer.js

```diff
@@ -43,3 +43,32 @@ export const determineNextPeriod = (currentPeriod, counter) => {
 
   return PERIODS.work;
 };
+export const displyNotification = (nextPeriod, currentPeriod) => {
+  const notificatinFunc = (text, icon) => new Notification('Pomodoro', {
+    body: text,
+    icon,
+  });
+  const showNotification = () => {
+    let text = null;
+    const icon = 'https://source.unsplash.com/1600x900/?nature,water;';
+    if (currentPeriod.id === 'work' && nextPeriod.id === 'long-brk') {
+      text = `Great job! Take a long break. You have ${PERIODS.longBrk.mins} minutes.`;
+      notificatinFunc(text, icon);
+    } else if (currentPeriod.id === 'short-brk' || currentPeriod.id === 'long-brk') {
+      text = `Time to get back to work! Your next break starts in ${PERIODS.work.mins} minutes.`;
+      notificatinFunc(text, icon);
+    } else if (currentPeriod.id === 'work') {
+      text = `Nice work! Take a short break. You have ${PERIODS.shortBrk.mins} minutes.`;
+      notificatinFunc(text, icon);
```

Comment:
Instead of calling `notificatinFunc` 3 times, you can simply set the value for `text` and `icon` inside your condition and only invoke the `notificatinFunc` after all your conditions. If you look at your code, this line is no different from line 59 and 56. Whenever you have the same line of code repeated exactly multiple times, then you know that you need to make a change.

Ask your peers for help if you get stuck on this one, and if they can't help you, you can come to me and I'll help.